lab_1_classify_by_unigrams/main.py

```diff
@@ -4,104 +4,71 @@
 """
 
 
-def tokenize(text: str) -> list[str] | None:
-    """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
+def tokenize(text):
+    text = text.lower()
+    for i in text:
+        if not i.isalnum() and i != ' ':
+            text = text.replace(i, '')
+    for i in text:
+        if i == ' ':
+            text = text.replace(' ','')
+    text = list(text)
+    return text
+tokens = tokenize(text)
+
+
+
+
+
+
+def calculate_frequencies(tokens):
+    dictionary = {}
+    explanation = {}
+    length = len(tokens)
+    for i in tokens:
+        dictionary[i] = (dictionary.setdefault(i, 0) + 1) / length
```

Comment:
are you sure this is correct calculation?

Reply:
I've changed it and now it should work properly